map/autoware_map_height_fitter/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@ cmake_minimum_required(VERSION 3.14)
 project(autoware_map_height_fitter)
 
 find_package(autoware_cmake REQUIRED)
```

Comment:
[Comment] I first considered why only this package's `CMakeLists.txt` has changes. But I understand this came from [this PR](https://github.com/autowarefoundation/autoware.universe/pull/10070/files).

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/10070#issue-2831777544

> This package fails to build with the autoware_utils 1.1.0. (This package depends on autoware_utils via autoware_lanelet2_extension)
> 
> Since autoware_utils is now depending on pcl as well, it overwrites some of PCL related CMAKE variables. In order to only import intended modules, PCL should be included after calling autoware_package().